### PR TITLE
feat(Sheet): multiline support for description

### PR DESCRIPTION
**Pull request by atabel**
jira: WEB-1559

### CODE REVIEW

File: src/sheet-root.tsx [hunk 1]
```diff
@@ -105,7 +105,8 @@ const showRadioListNativeSheet = ({
     (nativeSheetImplementation as NativeSheetImplementation)({
         title,
         subtitle,
-        description,
+        // TODO: add multiline support to native sheet
+        description: Array.isArray(description) ? description.join('\n') : description,
```
Comment (pladaria) [inline src/sheet-root.tsx:109]: perhaps, you should join with two line breaks, to simulate different paragraphs

Comment (pladaria) [reply src/sheet-root.tsx:109]: also note that:

`[].join('\n')` returns empty string, but in this case we probably want to set an `undefined` description.

Comment (atabel) [reply src/sheet-root.tsx:109]: I want to do some tests with the native implementation and see what happens with `\n` and `\n\n`

Comment (atabel) [reply src/sheet-root.tsx:109]: Tested. It seems to work quite fine with the double `\n\n`:
image

File: src/__stories__/sheet-story.tsx [hunk 2]
```diff
@@ -82,13 +82,20 @@ type SheetArgs = {
...
 type RadioListSheetArgs = SheetArgs & {
     selectedId: string;
 };
 
-export const RadioList: StoryComponent<RadioListSheetArgs> = ({title, subtitle, description, selectedId}) => {
+export const RadioList: StoryComponent<RadioListSheetArgs> = ({
+    title,
+    subtitle,
+    description,
+    multilineDescription,
```
Comment (yceballost) [inline src/__stories__/sheet-story.tsx:96]: multiParagraph name maybe fit better? 🤔

multilineDescription could be this:
image

**Review by yceballost — APPROVED** [review]

**Review by Marcosld — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

**Review by marcoskolodny — APPROVED** [review]

**Pull request merged**